src/material/bottom-sheet/bottom-sheet-container.ts

```diff
@@ -92,15 +92,17 @@ export class MatBottomSheetContainer extends CdkDialogContainer implements OnDes
     this._breakpointSubscription = breakpointObserver
       .observe([Breakpoints.Medium, Breakpoints.Large, Breakpoints.XLarge])
       .subscribe(() => {
-        this._toggleClass(
+        const classList = (this._elementRef.nativeElement as HTMLElement).classList;
+
```

Comment:
This change is unrelated to the fix, I just figured it's a good chance for a cleanup.